Q6.py

```diff
@@ -0,0 +1,12 @@
+def count_substring(string, substring):
+
+    #Using list comprehension and slicing , i have count the occcurence of substring in string. and then i have append 1 in list,
```

Comment:
You have extra space between `slicing and ,` the `slicing ,` in the comment. please remove that.

Reply:
Done sir